extensions/windows-event-log/tests/ConsumeWindowsEventLogTests.cpp

```diff
@@ -367,27 +372,19 @@ void batchCommitSizeTestHelper(int batch_commit_size, int expected_num_commits)
   test_plan->reset();
   LogTestController::getInstance().resetStream(LogTestController::getInstance().log_output);
 
-  {
-    reportEvent(APPLICATION_CHANNEL, "Event one");
-    reportEvent(APPLICATION_CHANNEL, "Event two");
-    reportEvent(APPLICATION_CHANNEL, "Event three");
-    reportEvent(APPLICATION_CHANNEL, "Event four");
-    reportEvent(APPLICATION_CHANNEL, "Event five");
-
-    test_controller.runSession(test_plan);
-
-    REQUIRE(LogTestController::getInstance().countOccurrences("processQueue commit") == expected_num_commits);
-  }
+  std::vector<std::string> events{"Event one", "Event two", "Event three", "Event four", "Event five"};
```

Comment:
Minor, but why not just a simple range based for here?

Reply:
Maybe I just wanted to keep it in a single line, but yea I see it's better to have a simple for loop here, fixed in [fa06429](https://github.com/apache/nifi-minifi-cpp/pull/891/commits/fa064298d99a1e0efc2609548062efd5f815085a)